cohort-evaluator-spark/src/test/resources/any-column/cql/MeasureAnyColumn-1.0.0.cql

```diff
@@ -0,0 +1,22 @@
+library "MeasureAnyColumn" version '1.0.0'
+using "CohortTest" version '0.0.1'
+
+codesystem "SNOMED": 'http://snomed.info/sct'
+codesystem "OTHER": 'http://system.other'
+code "MadeUpCode": 'cd1' from "SNOMED" display 'string1'
+code "OtherCode": 'cdA' from "OTHER"
+
+concept "MyConcept": {
+	"MadeUpCode", "OtherCode"
+} display 'MyConcept'
+
+context A
+
+define "cohort":
+  exists("Passes all checks")
+
+define function AnyColumn(key Choice<Any>, key String) returns List<Any> : external
```

Comment:
Maybe a conversation for the future, but would it be useful to publish a cql file that has our current collection of external functions?  It might be useful to have all of our external functions available in one place that can be imported easily, even though we only have 2 of them at the moment.

Reply:
Yea.  Something like `FHIRHelpers`.  Maybe `CohortHelpers`.

Reply:
I really like the idea of a CohortHelpers CQL library. I know there was some talk about creating a similar reusable CQL function library for use across Watson Health, not that those functions would necessarily go into CohortHelpers, but good to start capturing reusable functions (and logic?) in CQL libraries designed for reuse.

Reply:
moved to `CohortHelpers` in 3881ddb